src/AppConstants.js

```diff
@@ -59,3 +59,18 @@ export const FILTER_CATEGORIES = [
         ]
     }
 ];
+export const RISK_OF_CHANGE_LABEL = [ 'Very Low', 'Low', 'Moderate', 'High' ];
+export const RISK_OF_CHANGE_DESC = [
```

Comment:
Kinda feels like this should be an object, where the key is the label, and the description is the value

Reply:
Agreed, went this route to make progress as `var.1` was throwing an `Unexpected number` syuntax error.  Will be updating to an object and `var[1]` which doesn't complain.